distribution/lib/Standard/Base/0.0.0-dev/src/Error/File_Error.enso

```diff
@@ -20,6 +21,10 @@ type File_Error
     ## Indicates that a destination file already exists.
     Already_Exists (file : File)
 
+    ## Indicates that the user did not have the permission to complete the
+       requested operation on the given file.
+    Access_Denied (file : File)
```

Comment:
Would it be possible to say which operation is failing, or what permission we lack?

Reply:
I'm just wondering if in that case we should add this information to all other File errors - after all correlating a generic IO Error with the operation could also be useful. As well as Not Exists and other ones that come from Java.

Reply:
Yes, this might be a good future task.

Reply:
I created a task for it then #5828